packages/core/admin/shared/admin.ts

```diff
@@ -0,0 +1,12 @@
+interface StrapiFeaturesConfig {
+  /**
+   * These features are considered unstable and may have breaking changes within,
+   * use them at your own risk. They will likely be promoted to stable in the next
+   * major version. They do not follow semantic convention and we may remove them at any time
+   */
+  future?: {
+    contentReleases: boolean;
```

Comment:
Would make sense to use a interface so the contentReleases package can extend it instead of hardcoding it in the admin.

Reply:
Yes, now it's available in `@strapi/types` we can do that.